wp-content/themes/workingnyc/views/home.twig

```diff
@@ -38,21 +37,18 @@
             </a>
           </p>
         </div>
-      </div>
-    </div>
 
     {% if post.thumbnail %}
-    <div class="bg-image-grid hidden tablet:grid grid-cols-2 animated fadeIn">
-      <div></div>
 
-      <div class="bg-image-grid__mask-scale-1 relative">
-        <img class="bg-image-grid__img" loading="lazy" style="object-position: {{ post.thumbnail.focal_point }};" src="{{ post.thumbnail.src('desktop') }}" alt="{{ post.thumbnail.alt }}"/>
+      <div class="relative items-center hidden desktop:block tablet:block">
```

Comment:
`items-center` isn't working because the display is set to block (`desktop:block tablet:block`). The display should be set to flex instead so that `items-center` can work. 

I found this out by looking at the `items-center` class in Inspect Element (Developer Tools). There was this warning message: 

<img width="477" alt="image" src="https://github.com/CityOfNewYork/working-nyc/assets/8957837/cb54f054-3c5a-4706-8005-319dd385a9ab">